app/components/index.js

```diff
@@ -12,4 +13,5 @@ export {
   SingleProduct,
```

Comment:
there's something funny here. lines 11 & 12 looks like they have some kind of merge conflict. do you see what i'm seeing?

Reply:
Yeah, i also noticed but this is not appearing in the dev branch code but shows up as conflict with Dev. And without that it is working fine. Not sure if anything should be done here. i doubt.